tripleblindmarket/private-kit
# Connect Export To Staging Server & Redux Typescript

#### Description:
Connects export to Staging Server!
Adds connection to the new trusted-authorities repo for HA selection during export.
Next steps are to use that everywhere. 

Currently the HA yaml is hard coded (fine because this is all feature flagged still), this should ideally be a env variable to point to different yamls depending on lane. 

This also makes everything under the `/store` folder typescript, so that our data layer is less brittle.

Adds linking to privacy policy. Some design divergence (whole line is a link instead of just 'here') here to make this accessible across localization. We can revisit this and introduce the i18next `trans` component as a follow up if we want to specify a word across locals.

#### Screenshots:

Redux Store Shape:
![image (4)](https://user-images.githubusercontent.com/25315679/83930934-150f8f00-a768-11ea-8b21-b637ca11d14b.png)

<img width="527" alt="image" src="https://user-images.githubusercontent.com/25315679/83952077-345bfa00-a804-11ea-9914-741685668888.png">


#### How to test:
Go through the export flow! Using the staging server, create an access code for a case, and the upload process should work! There's a postman collection to make that easier if needed for creating)
